packages/manager/modules/pci/src/projects/project/kubernetes/details/node-pool/add/add.controller.js

```diff
@@ -41,17 +41,23 @@ export default class {
   create() {
     this.sendKubeTrack('details::nodepools::add::confirm');
 
+    const { flavor, name, antiAffinity } = this.nodePool;
+    const { monthlyBilling: monthlyBilled, autoscaling } = this.nodePool;
+    const { autoscale, nodes } = autoscaling;
+    const { lowest, desired, highest } = nodes;
```

Comment:
**suggestion:** Single de-structuring statement can be used.
```suggestion
    const {
      flavor,
      name,
      antiAffinity,
      monthlyBilling: monthlyBilled,
      autoscaling: {
        autoscale,
        nodes: {
          lowest, desired, highest
        }
      }
    } = this.nodePool;
```

Reply:
Thanks for advice. it's another way to do it.